finality-provider/service/app.go

```diff
@@ -682,3 +678,32 @@ func (app *FinalityProviderApp) metricsUpdateLoop() {
 		}
 	}
 }
+
+func (app *FinalityProviderApp) syncChainFpStatusLoop() {
```

Comment:
we should add a doc string saying that this loop will exit after at least one fp instance is started

Reply:
done at https://github.com/babylonlabs-io/finality-provider/pull/52/commits/502182975036de19d35550c2a14e73bc89c80644